### ISSUE TITLE
crash on starting new initialised project on windows 10 

### ISSUE THREAD

**Issue by JeyP90** [nuxt/framework#4186]
### Environment

------------------------------
- Operating System: `Windows_NT`
- Node Version:     `v17.0.0`
- Nuxt Version:     `3.0.0-27489107.7379184`
- Package Manager:  `npm@8.1.0`
- Builder:          `vite`
- User Config:      `-`
- Runtime Modules:  `-`
- Build Modules:    `-`
------------------------------

### Reproduction

- npx nuxi init nuxt3-app
- cd nuxt3-app
- npm i
- npm run dev

### Describe the bug

After upgrading my nuxt 3 version, the server is crashing after start. Tried to re-initialise a new project but same error.

### Logs

```shell
ERROR  [worker reload] [worker init] Only file and data URLs are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'                                        23:31:41

  at new NodeError (node:internal/errors:371:5)
  at defaultResolve (node:internal/modules/esm/resolve:983:11)
  at ESMLoader.resolve (node:internal/modules/esm/loader:422:30)
  at ESMLoader.getModuleJob (node:internal/modules/esm/loader:222:40)
  at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
  at link (node:internal/modules/esm/module_job:75:36)
```

**Comment by Jaffa-Cakes** [comment]
I'm having the same issue.

` ERROR  Only URLs with a scheme in: file, data are supported by the default ESM loader. On Windows, absolute paths must be valid file:// URLs. Received protocol 'c:'                                                                                                        13:45:45  

  at new NodeError (node:internal/errors:372:5)
  at throwIfUnsupportedURLScheme (node:internal/modules/esm/resolve:1075:11)
  at defaultResolve (node:internal/modules/esm/resolve:1155:3)
  at ESMLoader.resolve (node:internal/modules/esm/loader:604:30)
  at ESMLoader.getModuleJob (node:internal/modules/esm/loader:318:18)
  at ESMLoader.import (node:internal/modules/esm/loader:404:22)
  at importModuleDynamically (node:internal/modules/esm/translators:106:35)
  at importModuleDynamicallyCallback (node:internal/process/esm_loader:35:14)
  at prerender (/C:/Users/J/Documents/GitHub/trader/interface/node_modules/nitropack/dist/chunks/prerender.mjs:2142:39)
  at async /C:/Users/J/Documents/GitHub/trader/interface/node_modules/nuxt3/dist/index.mjs:1221:9`

node version `v17.9.0`

**Comment by danielroe** [comment]
Would you try with the latest Nuxt? I'm not getting this any more and it should have been resolved in https://github.com/nuxt/framework/pull/4187

**Comment by JeyP90** [comment]
> Would you try with the latest Nuxt? I'm not getting this any more and it should have been resolved in #4187.
> 
> Closing but I'll reopen if you still encounter the problem with a fresh install.

Thanks, thats fixed it for me. Had to remove the package-lock.json and node_modules and both my old and the fresh install is working again!

**Comment by tuxuuman** [comment]
I have a same problem

OS: ```Windows 10```
Node: ```v16.14.0```
Npm: ```8.5.4```
Nuxt: ```3.0.0-27494471.1091d45```

**Comment by mim289** [comment]
@tuxuuman - If you want a quick fix, if you pin to version `3.0.0-27488730.0cbc9cb` you should be good to at least get started!
